standard/standard_7.php

```diff
@@ -185,7 +185,7 @@ function pack ($format, $args = null, $_ = null) {}
  * @since 4.0
  * @since 5.0
  */
-function unpack ($format, $data, $offset) {}
+function unpack ($format, $data, $offset = null) {}
```

Comment:
[docs](http://php.net/manual/en/function.unpack.php) specify default value of the `$offset` as `0` rather then `null`

Reply:
Also it seems this function is already fixed in the earlier PR #522 :)